core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -264,9 +266,13 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
 
     // TODO (prashant) send conf instead of properties
     driverEndpoint = rpcEnv.setupEndpoint(
-      CoarseGrainedSchedulerBackend.ENDPOINT_NAME, new DriverEndpoint(rpcEnv, properties))
+        CoarseGrainedSchedulerBackend.ENDPOINT_NAME, createDriverEndpoint(properties))
   }
 
+  protected def createDriverEndpoint(
```

Comment:
This looks weird. How about the following (fits in 100 chars just fine);

      protected def createDriverEndpoint(properties: ArrayBuffer[(String, String)]): DriverEndpoint = {
        new DriverEndpoint(rpcEnv, properties)
      }